thefuck/rules/brew_cask_reinstall.py

```diff
@@ -0,0 +1,17 @@
+import re
+from thefuck.utils import for_app
+
+
+warning_regex = re.compile(r'Warning: Cask \'(?:.(?!is ))+\' is already installed.\n\n')
+message_regex = re.compile(r'To re-install (?:(?!, ).)+, run\n  `brew cask reinstall [^`]+`')
```

Comment:
Ditto. Should be changed:
```suggestion
message_regex = re.compile(r'To re-install (?:(?!, ).)+, run:\n  brew cask reinstall [^\n]+')
```